src/aphroditeInterfaceFactory.js

```diff
@@ -45,13 +48,44 @@ function separateStyles(stylesArray) {
   };
 }
 
+function stylesDiff(ltrStyles, rtlStyles) {
+  const styles = {}; // only includes the necessary styles and overrides
+  Object.keys(rtlStyles)
+    .filter(key => rtlStyles[key] !== ltrStyles[key])
+    .forEach((key) => {
+      let value = rtlStyles[key];
+      if (value && typeof value === 'object') {
+        value = stylesDiff(ltrStyles[key], value);
+      }
+      if (value !== null && typeof value !== 'undefined') styles[key] = value;
```

Comment:
Although this is the exact style are using all of my personal projects, both our style guide and performance would suggest using `== null` here.